test/vvenc_unit_test/vvenc_unit_test.cpp

```diff
@@ -276,6 +276,66 @@ static bool test_IntraPred()
 #endif // ENABLE_SIMD_OPT_INTRAPRED
 
 #if ENABLE_SIMD_TRAFO
+static bool check_cpyCoeff( TCoeffOps* ref, TCoeffOps* opt, unsigned num_cases, unsigned W )
+{
+  DimensionGenerator dim;
+
+  static constexpr size_t buf_size = MAX_CU_SIZE * MAX_CU_SIZE;
+
+  // Use xMalloc to create aligned buffers for x86.
+  Pel    *src     = ( Pel*    )xMalloc( Pel,    buf_size );
+  TCoeff *dst_ref = ( TCoeff* )xMalloc( TCoeff, buf_size );
+  TCoeff *dst_opt = ( TCoeff* )xMalloc( TCoeff, buf_size );
+
+  bool passed = true;
+
+  InputGenerator<Pel> inp_gen{ 10, /*is_signed=*/false }; // unsigned 10-bit
+
+  std::ostringstream sstm_test;
+  sstm_test << "TCoeffOps::cpyCoeff" << W;
+  std::cout << "Testing " << sstm_test.str() << std::endl;
+
+  for( unsigned n = 0; n < num_cases; n++ )
+  {
+    unsigned  height    = dim.get( W, MAX_CU_SIZE );
+    unsigned  width     = dim.get( W, MAX_CU_SIZE, W ); // Width must be a multiple of W.
```

Comment:
I think that both width and height can only be powers of 2 (including `1` and `2`, because of ISP). I guess we handle the w==1 and w==2 manually and the generalization to multiples of 4 doesnt matter. But if later on someone writes a more strict specialization it would fail the test while being correct.